src/views/Home.vue

```diff
@@ -494,10 +494,34 @@ export default class Home extends Vue {
     }
   }
 
-  refreshRecentDatasetDetails() {
-    for (const d of this.datasetViews) {
-      this.girderResources.getFolder(d.datasetId);
-      this.girderResources.getCollection(d.configurationId);
+  async refreshRecentDatasetDetails() {
+    if (this.datasetViews.length === 0) return;
```

Comment:
I think this line is preventing the function from working

Reply:
I reverted to previous behavior.